app/services/ecr_service.rb

```diff
@@ -0,0 +1,49 @@
+class EcrService
+  def validate_image!(heritage)
+    @heritage = heritage
```

Comment:
Please initialize the instance variable in a constructor

```suggestion
  def initialize(heritage)
    @heritage = heritage
  end

  def validate_image!(heritage)
```